docs/content/node-operation/full-observer-node.mdx

```diff
@@ -14,21 +14,26 @@ node for local queries without the need for staking.
 
 ## Running a node
 
-The full observer node was built in collaboration with the `optakt` team. The
-primary way to run a full observer node is with the Flow Data Provisioning
-Service (DPS).
+The full observer node was built in collaboration with the Optakt team.
+The primary way to run a full observer node is with the Flow Data Provisioning Service (DPS).
 
 ### Data Provisioning Service
 
-The [Data Provisioning Service](https://github.com/optakt/flow-dps/) provides
-efficient access to the full history of flow blockchain data from a single API.
-DPS was built by the `optakt` team and greatly streamlines use cases that require
-archival data.
+The [Flow DPS](https://github.com/optakt/flow-dps/) provides efficient access to the full history
+of Flow blockchain data from a single API.
+The Flow DPS was built by the [Optakt](https://www.optakt.io/) team and greatly streamlines 
+use cases that require archival data.
+It provides a scalable and efficient way to access the history of the Flow execution state,
+both for current live [sporks](/node-operation/spork/) and for [past sporks](/node-operation/past-sporks).
```

Comment:
The sentence and the sentence on line 26 say pretty much the same thing. Maybe change this section to
```suggestion
The [Flow DPS](https://github.com/optakt/flow-dps/) was built by the [Optakt](https://www.optakt.io/) team and greatly streamlines use cases that require archival data.
It provides a scalable and efficient way to access the history of the Flow execution state,
both for current live [sporks](/node-operation/spork/) and for [past sporks](/node-operation/past-sporks).
```

Reply:
I agree that it was a bit redundant. I updated the phrasing.

Reply:
@peterargue I agree, but that was written before we made the suggestion, so we tried to be minimally invasive ;)